src/index.css

```
@@ -378,20 +390,24 @@
   ::-webkit-scrollbar {
     @apply h-[5px] w-[5px];
   }
+
   ::-webkit-scrollbar-track {
     @apply bg-transparent;
   }
+
   ::-webkit-scrollbar-thumb {
     @apply rounded-[5px];
     background-color: var(--scroller-color);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)